src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -15,6 +17,8 @@ interface CommandArgs {
 interface Options extends GlobalOptions {
   period?: string;
   date?: string;
+  outputFile?: string;
+  output?: string;
```

Comment:
There is no need to introduce new output option since it is already introduced by the parent class, you just have to reuse it. See https://github.com/pnp/office365-cli/pull/1037/files

Reply:
I have fixed this in the latest commit.